### DISCUSSION TITLE
Re: Quadlock Carplay Adapter exact same as carlinkit 3.0

### DISCUSSION THREAD

**Discussion by CollapsedLegacy** [ludwig-v/wireless-carplay-dongle-reverse-engineering#165] [Q&A]
Hi, I know this is for carlinkit's products however, quadlock has the same exact components inside just different software. I have an issue where it doesn't show up Bluetooth or wifi login so I cannot go to 192.168.50.2 and I can't access carplay wirelessly. I want to know if there's a way to make their software work with this?

**Comment by kaliv79** [comment]
I have the same problem I wrote directly to them

**Comment by Zubastic** [comment]
They are just OEM resellers of carlinkit.
There several of them:

- WIZCAR
- NAVIPILOT
- MyCarPlay
- etc

How detect them:

1. About 70% of them having wifi/bluetooth with name Autobox
2. All of them having connection to 192.168.50.2 (this is red flag)
3. All of them having same design of web setting page

They all using http://ab720907.link:8081 as web point. So here no difference between them. Difference only in hardware type.
Here response from box:
```json
	"BoxInfo": {
		"bt": "AutoBox-2222",
		"wifi": "AutoBox-1111",
		"boxName": "",
		"MDLinkType": "",
		"HULinkType": "",
		"MDModel": "",
		"MDOSVersion": "",
		"MDLinkVersion": "",
		"hardwareVer": "YMYE-WR2C-0212",
		"ver": "2024.07.31.1438",
		"cgiVer": "Jul 30 2024 15:08:50",
		"mfd": "...",
		"uuid": "...",
		"productType": "UC2WD_AUTOKIT",
		"mac": "...",
		"wifiChipType": 6,
		"customVer": 2,
		"customCode": "",
		"platformType": "M6",
		"fileName": "UC2WD_AUTOKIT_Update.img",
		"boxType": "YC",
		"logFileSuffix": "YC2WD",
		"tabId": "10",
		"code": "...",
		"needActive": 0
	}
```

Here main info in `productType`, `wifiChipType` and `customVer`.

**Comment by Zubastic** [comment]
- U2IW
- U2W
- U2HW
- U2HC
- U2AW
- U2O
- U2AC
- UC2W
- C2W
- UC2WD
- UC2D
- UC2HW
- UC2HWD
- UC2HC
- UC2HCD
- UC2AW
- C2AW
- UC2CA
- O2W
- A15H
- A15W

This is all types of boxes with same firmware.
